pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -91,7 +93,8 @@ public MultiStageBrokerRequestHandler(PinotConfiguration config, BrokerRoutingMa
         CalciteSchemaBuilder.asRootSchema(new PinotCatalog(tableCache)),
         new WorkerManager(_reducerHostname, _reducerPort, routingManager));
     _queryDispatcher = new QueryDispatcher();
-    _mailboxService = new GrpcMailboxService(_reducerHostname, _reducerPort, config);
+    _mailboxService = new MultiplexingMailboxService(new GrpcMailboxService(_reducerHostname, _reducerPort, config),
+        new InMemoryMailboxService(_reducerHostname, _reducerPort));
```

Comment:
multistage broker request handler shouldn't know about this. make this a multiplexingmailboxservice builder pattern
- e.g.
```
_mailboxService = MultiplexingMailboxService.newInstance(pinotConfig)

// in MultiplexingMailboxService
public static MultiplexingMailboxService newInstance(PinotConfiguration pinotConfig) {
  builder = MultiplexingMailboxService.Buidler();
  if (pinotConfig.contains(ENABLE_IN_MEMORY_MAILBOX)) {
    builder.add(new InMemoryMailboxService());
  }
  if (pintoConfig.contains(ENABLE_GRPC_MAILBOX)) {
    builder.add(new GRPCMailboxService());
  }
  return builder.build();
}
```